### ISSUE TITLE
pin mini player 

### ISSUE THREAD

**Issue by loaykhalifa** [Kentakoong/MusicBar#26]
is there is way to pin mini player ( or you can make it as widget )
Thanks for first & only app capture audio for every app on mac ❤️

**Comment by Kentakoong** [comment]
widget would be cool, actually have a plan for that when I have time for development, trying to rollout v2 UI right now, but the settings still kinda looks ugly, that's why.

For the miniplayer, could you elaborate more, do you mean like, make it always open to see the miniplayer all the time, something like that? (if it is, let me research, because the menubar itself is based on NSMenu so I'm not sure if it has that capability or not, or else i have to switch things up to make it possible.)

**Comment by loaykhalifa** [comment]
Yes, always show the benefit of sharing your widget or mini-player on OBS or other streaming software. With the new 'Now Play API,' every DJ using different software can now share what’s playing live on streaming platforms. This seamless integration enhances the viewer experience by providing real-time updates on the current track.

**Comment by loaykhalifa** [comment]
![CleanShot 2024-12-13 at 22 26 33](https://github.com/user-attachments/assets/f6785b6b-7d85-46bf-a5fc-f320c2a5c5e8)

other suggestion :
share this info as url link 👍 would be awesome

**Comment by Kentakoong** [comment]
> ![CleanShot 2024-12-13 at 22 26 33](https://private-user-images.githubusercontent.com/102491821/395688573-f6785b6b-7d85-46bf-a5fc-f320c2a5c5e8.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQxOTU5MzYsIm5iZiI6MTczNDE5NTYzNiwicGF0aCI6Ii8xMDI0OTE4MjEvMzk1Njg4NTczLWY2Nzg1YjZiLTdkODUtNDZiZi1hNWZjLWYzMjBjMmE1YzVlOC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxNFQxNzAwMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYTRlYjQyMDM3OWQ3OTgwN2UwZjcxYmQ2ZGQ2ZWJkMWQ5ZDhkODVjZGMzMWNhOGMzOTYzODNjNjczYmExYjU5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Akl7lhsQC3FY4t81W0yUACSNP2MTNGmlbNvD54timGE)
> 
> other suggestion : share this info as url link 👍 would be awesome

hmmm, that would be quite hard, since it's using apple's private api, and it doesn't provide the link to the stuff that we're interacting with, but let me see

**Comment by loaykhalifa** [comment]
thanks @Kentakoong

**Comment by loaykhalifa** [comment]
Please @Kentakoong  check out this [project](https://www.nowplaying2.com/tutorials/setting-up-obs)

**Comment by Kentakoong** [comment]
> Please @Kentakoong check out this [project](https://www.nowplaying2.com/tutorials/setting-up-obs)

for this, i could expose some sorts of API, I'll keep in mind of this because it's a priority feature for me too

**Comment by loaykhalifa** [comment]
there is other project [Sleeve](https://apps.apple.com/us/app/sleeve-for-spotify-music/id1606145041)
but it doesn’t have what you did for all app api

is there any sooner update ?!!

**Comment by Kentakoong** [comment]
let me work on the new call system real quick, and I'll hop on this
